doc/source/architecture/switch-config.rst

```diff
@@ -632,7 +632,10 @@ We assume the following L3 related config to be preconfigured on each device, ex
       route-target import evpn 65130:4102
       redistribute connected route-map RM-CC-CLOUD02-REDIST
       redistribute static route-map RM-CC-CLOUD02-REDIST
-   
+  
+  # Octa crashes when `redistribute static` is configured but no static route is present
+  # Let us configure a dummy route from the zeroconf range to hackaround that problem
+  ip route vrf CC-CLOUD02 169.254.255.255 255.255.255.255 null0   
```

Comment:
This is boilerplate config and nothing the driver needs to configure, right?

Reply:
No, it's in the preconfigured part.